src/Controls/DualScreen/src/Controls.DualScreen.csproj

```diff
@@ -1,30 +1,41 @@
-<Project Sdk="MSBuild.Sdk.Extras">
+﻿<Project Sdk="Microsoft.Net.Sdk">
   <PropertyGroup>
-    <RootNamespace>Microsoft.Maui.Controls.DualScreen</RootNamespace>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;net6.0;$(MauiPlatforms)</TargetFrameworks>
```

Comment:
can you use this?
`netstandard2.1;netstandard2.0;$(_MauiDotNetTfm);$(MauiPlatforms)`

Reply:
Yes, done!